project/profiles/capacity4more/capacity4more.profile

```diff
@@ -85,8 +85,8 @@ function capacity4more_setup_set_variables(&$install_state) {
     'theme_default' => 'kapablo',
     'admin_theme' => 'seven',
     'node_admin_theme' => 0,
-    'jquery_update_jquery_version' => '2.1',
-    'jquery_update_jquery_admin_version' => '2.1',
+//    'jquery_update_jquery_version' => '2.1',
+//    'jquery_update_jquery_admin_version' => '2.1',
     'page_manager_node_view_disabled' => FALSE,
```

Comment:
@ybaras 
just set _jquery_update_jquery_version_ like:
```PHP
'jquery_update_jquery_version' => '1.8'
```

better to be explicit.